engine/src/main/java/com/redhat/cloud/notifications/processors/email/connector/dto/EmailNotification.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.cloud.notifications.processors.email.connector.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.cloud.notifications.models.Template;
+
+import java.util.Collection;
+
+/**
+ * Represents the data structure that the email connector is expecting.
+ * @param subscribers the list of user {@link java.util.UUID}s which were
+ *                    subscribed to the event.
+ * @param action the action from the event that generated this whole process.
+ * @param bodyTemplate the non-compiled, non-rendered body's template for the
+ *                     email.
+ * @param subjectTemplate the non-compiled, non-rendered subject's template for
+ *                        the email.
+ */
+public record EmailNotification(
+        @JsonProperty("subscribers")            Collection<String> subscribers,
+        @JsonProperty("recipient_settings")     Collection<RecipientSettings> recipientSettings,
+        @JsonProperty("action")                 String action,
+        @JsonProperty("body_template")          Template bodyTemplate,
+        @JsonProperty("subject_template")       Template subjectTemplate,
```

Comment:
This is no longer needed, right? Instead we should be passing the rendered email `subject` and `body`.

Reply:
Right! Removed!